flight/PiOS/Common/pios_bmp280.c

```diff
@@ -0,0 +1,441 @@
+/**
+ ******************************************************************************
+ * @addtogroup PIOS PIOS Core hardware abstraction layer
+ * @{
+ * @addtogroup PIOS_BMP280 BMP280 Functions
+ * @brief Hardware functions to deal with the altitude pressure sensor
+ * @{
+ *
+ * @file       pios_bmp280.c
+ * @author     Tau Labs, http://taulabs.org, Copyright (C) 2012-2014
```

Comment:
Was this ever part of TL? If not, I would just put dRonin as author. Also, don't forget the additional dRonin notice (see license template). Also applies to other BMP280 files.